css/components/navbar.css

```diff
@@ -0,0 +1,100 @@
+.nav__bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 3.25rem;
+}
+
+.nav__logo__container {
+  margin: 0 1.25rem 0 0;
+}
+
+.nav__logo__img {
+  height: 3.25rem;
+  width: auto;
+}
+
+.nav__toggle__button span {
+  width: 3.25rem;
+  font-size: 2rem;
+  text-align: center;
+  padding: 0;
```

Comment:
if you need to set padding as 0 on a span, then, definitely there is something wrong